modin/pandas/accessor.py

```diff
@@ -208,7 +215,9 @@ class ModinAPI:
         Object to operate on.
     """
 
-    def __init__(self, data):
+    _data: Union[DataFrame, Series]
```

Comment:
Why did this become a class attribute?

Reply:
This is a practice that allows to specify exactly what attributes all instances have and, in addition, makes it possible to specify its type.

What is written here applies to all of your similar comments.